CommonClient.py

```diff
@@ -356,6 +357,26 @@ async def update_death_link(self, death_link: bool):
         if old_tags != self.tags and self.server and not self.server.socket.closed:
             await self.send_msgs([{"cmd": "ConnectUpdate", "tags": self.tags}])
 
+    def gui_error(self, title: str, text: typing.Union[Exception, str]):
+        """Displays an error messagebox"""
+        if not self.ui or not title or not text:
```

Comment:
no title/text should probably not be silently ignored?

Reply:
Oh, yeah, forgot to clean that up :/
We have to handle bad arguments because throwing from there is hard to track since it may crash the GUI. Text already got the `str(text)` treatment, title will have to become `title or 'Error'`